src/com/palmergames/bukkit/towny/TownyFormatter.java

```diff
@@ -644,55 +663,59 @@ public static StatusScreen getStatus(TownyWorld world, CommandSender sender) {
 	 * Utility methods used in the Status Screens.
 	 */
 	
-	private static String colourKeyValue(String key, String value) {
-		return String.format(keyValueFormat, Translation.of("status_format_key_value_key"), key, Translation.of("status_format_key_value_value"), value); 
+	private static Component colourKeyValue(String key, String value) {
+		return TownyComponents.miniMessage(String.format(keyValueFormat, Translation.of("status_format_key_value_key"), key, Translation.of("status_format_key_value_value"), value));
+	}
+
+	private static Component colourKeyValue(String key, Component value) {
+		return TownyComponents.miniMessage(String.format(keyValueFormat, Translation.of("status_format_key_value_key"), key, Translation.of("status_format_key_value_value"), TownyComponents.unMiniMessage(value)));
 	}
 	
-	private static String colourKey(String key) {
-		return String.format(keyFormat, Translation.of("status_format_key_value_key"), key); 
+	private static Component colourKey(String key) {
+		return TownyComponents.miniMessage(String.format(keyFormat, Translation.of("status_format_key_value_key"), key));
 	}
 	
-	private static String colourKeyImportant(String key) {
-		return String.format(keyFormat, Translation.of("status_format_key_important"), key);
+	private static Component colourKeyImportant(String key) {
+		return TownyComponents.miniMessage(String.format(keyFormat, Translation.of("status_format_key_important"), key));
 	}
 	
-	private static String colourBracketElement(String key, String value) {
-		return String.format(bracketFormat, Translation.of("status_format_bracket_element"), key, value);
+	private static Component colourBracketElement(String key, String value) {
+		return TownyComponents.miniMessage(String.format(bracketFormat, Translation.of("status_format_bracket_element"), key, value));
 	}
 	
-	private static String colourHoverKey(String key) {
-		return String.format(hoverFormat, Translation.of("status_format_hover_bracket_colour"), Translation.of("status_format_hover_key"), key, Translation.of("status_format_hover_bracket_colour"));
+	private static Component colourHoverKey(String key) {
+		return TownyComponents.miniMessage(String.format(hoverFormat, Translation.of("status_format_hover_bracket_colour"), Translation.of("status_format_hover_key"), key, Translation.of("status_format_hover_bracket_colour")));
 	}
 	
-	private static String formatPopulationBrackets(int size) {
-		return String.format(" %s[%s]", Translation.of("status_format_list_2"), size);

```

Comment:
The String-returning methods are used in at least one other side-plugin and should be readded and deprecated.

Reply:
The methods are all private, they're only used by us in the TownyFormatter class.

Reply:
Ah, I might have just replicated those methods in a side-plugin then.